minio/api.py

```diff
@@ -365,6 +365,8 @@ def get_bucket_policy(self, bucket_name, prefix=""):
         if not policy_dict:
             return policy.Policy.NONE
 
+        if not policy_dict.get('Statement'):
```

Comment:
This is not the right fix.

```py
>>> policy_dict = {'Statement': []}
>>> print not policy_dict.get('Statement')
True
>>> policy_dict = {'Statement': None}
>>> print not policy_dict.get('Statement')
True
```

We don't need to throw error on empty statement list.